### PR TITLE
Add Youtube videos

### PR DESCRIPTION
**Pull request by cougil**

### CODE REVIEW

**Comment by abelsromero** [comment]
I'd be nice to show which talks have a video in the talks list

File: 2018/_data/talks.json [hunk 1]
```diff
@@ -129,6 +131,8 @@
         "best practices"
       ],
       "level": "beginner",
+      "file": "JBCN_DIA 12/SALA 13/180612_JBCN_RUSTAM MEHMANDAROV_13_1.mp4",
```
Comment (jonathanvila) [inline 2018/_data/talks.json]: What's the purpose of the "file" property ? I haven't seen anywhere its usage

Comment (cougil) [reply 2018/_data/talks.json]: It was used the upload videos script, I used this field for that, otherwise the videos could not be uploaded ;-)

**Comment by cougil** [comment]
Changes applied! @abelsromero ;-)

**Pull request merged**